utils/node_local_test.py

```diff
@@ -3231,6 +3233,142 @@ def test_daos_fs_check(self):
         if dir_nr != 12:
             raise NLTestFail('Wrong number of sub-directories in lost+found')
 
+    def test_daos_fs_fix(self):
+        """Test DAOS FS Fix Tool"""
+
+        pool = self.pool.id()
+        cont = self.container
+
+        dfuse = DFuse(self.server,
+                      self.conf,
+                      pool=self.pool.id(),
+                      container=self.container,
+                      caching=False)
+        dfuse.start(v_hint='fs_fix_test', log_mask='CRIT')
```

Comment:
This is going to have the effect of disabling the log analysis leak checking which isn't desireable however the code still runs under valgrind so that should still catch any errors.